src/main/java/productDependency/CreateOrderPage.java

```diff
@@ -0,0 +1,1481 @@
+    /**
+     * Method to select Period.
+     * @throws IOException 
+     */
+    public void selectPeriod5() throws IOException{
+    	CreateOrderPage sp=new CreateOrderPage(driver);
+        WebElement Periodelement = driver.findElement(By.xpath("//select[@name='period']"));
+        Select se = new Select(Periodelement);
+        se.selectByVisibleText(BasePage.getCellData(xlsxName, sheetName, 4, 2));
+
+    }
+    
+
+    private WebElement selectOrderType;
+    /**
+     * Method to select Order Type.
+     * @throws IOException 
+     */
+    public void selectOrderType() throws IOException{
+    	CreateOrderPage sp=new CreateOrderPage(driver);
+        WebElement OrderTypeelement = driver.findElement(By.xpath("//select[@name='billingTypeId']"));
+        Select se = new Select(OrderTypeelement);
+        se.selectByVisibleText(BasePage.getCellData(xlsxName, sheetName, 5, 0));
+
+    }
+    
+    private WebElement selectOrderType1;
+    /**
+     * Method to select Order Type.
+     * @throws IOException 
+     */
+    public void selectOrderType1() throws IOException{
+    	CreateOrderPage sp=new CreateOrderPage(driver);
+        WebElement OrderTypeelement = driver.findElement(By.xpath("//select[@name='billingTypeId']"));
+        Select se = new Select(OrderTypeelement);
+        se.selectByVisibleText(BasePage.getCellData(xlsxName, sheetName, 5, 0));
+
+    }
+    
+    private WebElement selectOrderType2;
+    /**
+     * Method to select Order Type.
+     * @throws IOException 
+     */
+    public void selectOrderType2() throws IOException{
+    	CreateOrderPage sp=new CreateOrderPage(driver);
+        WebElement OrderTypeelement = driver.findElement(By.xpath("//select[@name='billingTypeId']"));
+        Select se = new Select(OrderTypeelement);
+        se.selectByVisibleText(BasePage.getCellData(xlsxName, sheetName, 5, 0));
+
+    }
+    
+    private WebElement selectOrderType3;
+    /**
+     * Method to select Order Type.
+     * @throws IOException 
+     */
+    public void selectOrderType3() throws IOException{
+    	CreateOrderPage sp=new CreateOrderPage(driver);
+        WebElement OrderTypeelement = driver.findElement(By.xpath("//select[@name='billingTypeId']"));
+        Select se = new Select(OrderTypeelement);
+        se.selectByVisibleText(BasePage.getCellData(xlsxName, sheetName, 5, 1));
+
+    }
+    
+    private WebElement selectOrderType5;
+    /**
+     * Method to select Order Type.
+     * @throws IOException 
+     */
+    public void selectOrderType5() throws IOException{
+    	CreateOrderPage sp=new CreateOrderPage(driver);
+        WebElement OrderTypeelement = driver.findElement(By.xpath("//select[@name='billingTypeId']"));
+        Select se = new Select(OrderTypeelement);
+        se.selectByVisibleText(BasePage.getCellData(xlsxName, sheetName, 5, 1));
+
+    }
+    
+    private WebElement selectOrderType4;
+
+	/**
+	 * Method to select Order Type.
+	 * 
+	 * @throws IOException
+	 */
```

Comment:
@mayankchhajer Remove all the unused WebElement for the methods select Period and Select Ordertype